test/source/tests/browser-unit-tests/unit-ContactStore.js

```diff
@@ -357,3 +357,35 @@ BROWSER_UNIT_TEST_NAME(`ContactStore gets a valid pubkey by e-mail, or exact pub
   }
   return 'pass';
 })();
+
+BROWSER_UNIT_TEST_NAME(`ContactStore stores postfixed fingerprint internally for X.509 certificate`);
```

Comment:
Thank you for these tests. This particular test is testing our specific implementation rather then the general purpose - the more general purpose is that OpenPGP and S/MIME keys don't conflict with one another in storage, which would make a slightly more meaningful test.

But this is really a nitpick, not worth changing.

Reply:
Yes, I was thinking about creating an OpenPGP key having the same RSA public key as S/MIME. It's not quite straightforward to perform but in theory possible. Would you want me to spend time on this? Or we can simply add a test checking that OpenPGP keys' fingerprints are not suffixed in the storage?

Reply:
Not needed - this is a bit of an esoteric situation that would be better to leave for later. I'll file it as an issue for the future (who knows - maybe our S/MIME integration will work that way - but not right now).